src/stylesheets/core/_utilities.scss

```diff
@@ -26,22 +26,12 @@
 // Unstyled list helper
 
 @mixin unstyled-list() {
-  display: block;
+  @include margin(0 null);
```

Comment:
Note: it's not undoing our margins (we did not add any margins for `ul`s), but webkits margins. You'll find:
```
-webkit-margin-before: 1em;
-webkit-margin-after: 1em;
```
applied to `ul` in the user agent styles. Which makes me think, should we add these in to be more explicit?